giskard_vision/landmark_detection/detectors/ethnicity_bias_detector.py

```diff
@@ -0,0 +1,34 @@
+from giskard_vision.landmark_detection.dataloaders.wrappers import (
+    CachedDataLoader,
+    EthnicityDataLoader,
+    FilteredDataLoader,
+)
+
+from .base import LandmarkDetectionBaseDetector
+from .decorator import maybe_detector
+
+
+@maybe_detector("ethnicity_landmark", tags=["landmark"])
+class EthnicityDetectorLandmark(LandmarkDetectionBaseDetector):
+    """
+    Detector that evaluates models performance depending on ethnicity
+    """
+
+    group: str = "Ethical"
+
+    def get_dataloaders(self, dataset):
+        ethnicity_dl = EthnicityDataLoader(dataset, ethnicity_map={"indian": "asian"})
+        cached_dl = CachedDataLoader(ethnicity_dl, cache_size=None, cache_img=False, cache_marks=False)
+
+        ethnicities = [("white ethnicity", self._white_ethnicity)]
+
+        dls = []
+
+        for e in ethnicities:
+            current_dl = FilteredDataLoader(cached_dl, e[1])
+            dls.append(current_dl)
+
+        return dls
+
+    def _white_ethnicity(self, elt):
```

Comment:
let's automate the creation of these functions based on: https://github.com/Giskard-AI/giskard-vision/blob/285a948d6e2f5c91a99fd14adc8f78b6239a8750/giskard_vision/landmark_detection/dataloaders/wrappers.py#L476C5-L476C26